### PR TITLE
Property discriminators

### PR DESCRIPTION
**Pull request by ksuderman**
Adds:

- properties can now have discriminators generated 
- custom XSD datatypes can be defined in the DSL to generate a Datatype.xsd file
- some deployment tweaks
- examples for morphological information

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,10 @@
+#VERSION=$(shell cat VERSION)
+VERSION=1.3.0
```
Comment (keighrim) [inline Makefile]: which one is supposed to be the correct one to use? (`VERSION` file still exists)

Comment (ksuderman) [reply Makefile]: Fixed.  The version number defined in the VERSION file is the one used.

File: lapps.vocabulary [hunk 2]
```diff
@@ -195,6 +214,45 @@ NamedEntity {
     }
 }
 
+Morphology {
```
Comment (reckart) [inline lapps.vocabulary]: Would it be useful to sync this with http://universaldependencies.org/u/feat/index.html ?

**Comment by nancyide** [comment]
We’ve already discussed with Jan Hajic about this, so yes!

**Comment by reckart** [comment]
Great :) Then (I hope) we shall be compatible. DKPro Core tries to follow the UD features (https://dkpro.github.io/dkpro-core/releases/1.10.0/docs/typesystem-reference.html#_morphology) - although I have to check if they have made changes since the last time I updated the type system...

**Comment by reckart** [comment]
Well... yes, they have. Do you have any idea what kind of scheme they follow when evolving their morphological features? If we sync with them and they keep changing things, its could be causing compatibility problems.

**Comment by nancyide** [comment]
I spoke with Jan about this—they don’t really have any versioning system, but they recognized the need. I’ll ping him again (cc you) and see where things stand.

**Pull request merged**